resources/shared/aliases.yml

```diff
@@ -0,0 +1,1351 @@
+--- 
+:aliases: 
+  :language: 
+    :aa_SAAHO: ssy
+    :aar: aa
+    :abk: ab
+    :afr: af
+    :aju: jrb
+    :aka: ak
+    :alb: sq
+    :als: sq
+    :amh: am
+    :ara: ar
+    :arb: ar
+    :arg: an
+    :arm: hy
+    :art_lojban: jbo
+    :asm: as
+    :ava: av
+    :ave: ae
+    :aym: ay
+    :ayr: ay
+    :aze: az
+    :azj: az
+    :bak: ba
+    :bam: bm
+    :baq: eu
+    :bcc: bal
+    :bcl: bik
+    :bel: be
+    :ben: bn
+    :bih: bh
+    :bis: bi
+    :bod: bo
+    :bos: bs
+    :bre: br
+    :bul: bg
+    :bur: my
+    :bxk: luy
+    :bxr: bua
+    :cat: ca
+    :ces: cs
+    :cha: ch
+    :che: ce
+    :chi: zh
+    :chu: cu
+    :chv: cv
+    :cld: syr
+    :cmn: zh
+    :cor: kw
+    :cos: co
+    :cre: cr
+    :cwd: cr
+    :cym: cy
+    :cze: cs
+    :dan: da
+    :deu: de
+    :dgo: doi
+    :dhd: mwr
+    :dik: din
+    :diq: zza
+    :div: dv
+    :dut: nl
+    :dzo: dz
+    :ekk: et
+    :ell: el
+    :emk: man
+    :eng: en
+    :epo: eo
+    :esk: ik
+    :est: et
+    :eus: eu
+    :ewe: ee
+    :fao: fo
+    :fas: fa
+    :fat: ak
+    :fij: fj
+    :fin: fi
+    :fra: fr
+    :fre: fr
+    :fry: fy
+    :fuc: ff
+    :ful: ff
+    :gaz: om
+    :gbo: grb
+    :geo: ka
+    :ger: de
+    :gla: gd
+    :gle: ga
+    :glg: gl
+    :glv: gv
+    :gno: gon
+    :gre: el
+    :grn: gn
+    :gug: gn
+    :guj: gu
+    :gya: gba
+    :hat: ht
+    :hau: ha
+    :hbs: sr_Latn
+    :hdn: hai
+    :hea: hmn
+    :heb: he
+    :her: hz
+    :him: srx
+    :hin: hi
+    :hmo: ho
+    :hrv: hr
+    :hun: hu
+    :hye: hy
+    :i_ami: ami
+    :i_bnn: bnn
+    :i_hak: hak
+    :i_klingon: tlh
+    :i_lux: lb
+    :i_navajo: nv
+    :i_pwn: pwn
+    :i_tao: tao
+    :i_tay: tay
+    :i_tsu: tsu
+    :ibo: ig
+    :ice: is
+    :ido: io
+    :iii: ii
+    :ike: iu
+    :iku: iu
+    :ile: ie
+    :in: id
+    :ina: ia
+    :ind: id
+    :ipk: ik
+    :isl: is
+    :ita: it
+    :iw: he
+    :jav: jv
+    :ji: yi
+    :jpn: ja
+    :jw: jv
+    :kal: kl
+    :kan: kn
+    :kas: ks
+    :kat: ka
+    :kau: kr
+    :kaz: kk
+    :khk: mn
+    :khm: km
+    :kik: ki
+    :kin: rw
+    :kir: ky
+    :kmr: ku
+    :knc: kr
+    :kng: kg
+    :knn: kok
+    :kom: kv
+    :kon: kg
+    :kor: ko
+    :kpv: kv
+    :kua: kj
+    :kur: ku
+    :lao: lo
+    :lat: la
+    :lav: lv
+    :lbk: bnc
+    :lim: li
+    :lin: ln
+    :lit: lt
+    :ltz: lb
+    :lub: lu
+    :lug: lg
+    :lvs: lv
+    :mac: mk
+    :mah: mh
+    :mal: ml
+    :mao: mi
+    :mar: mr
+    :may: ms
+    :mhr: chm
+    :mkd: mk
+    :mlg: mg
+    :mlt: mt
+    :mnk: man
+    :mo: ro_MD
+    :mol: ro_MD
+    :mon: mn
+    :mri: mi
+    :msa: ms
+    :mup: raj
+    :mya: my
+    :nau: na
+    :nav: nv
+    :nbl: nr
+    :nde: nd
+    :ndo: ng
+    :nep: ne
+    :nld: nl
+    :nno: nn
+    ? !ruby/symbol "no"
```

Comment:
Not completely sure... probably because of this:

```ruby
YAML.load("no")  # => false
```

Although this works:

```ruby
YAML.load(":no")  # => :no
```